.github/workflows/build.yml

```diff
@@ -11,6 +11,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Install system dependencies
+        run: | 
+         sudo apt-get update  -y 
```

Comment:
nit: a comment here giving more context around why we are installing gcc would be helpful